cmd/flipt/main.go

```diff
@@ -134,13 +151,14 @@ func main() {
 	banner = buf.String()
 
 	rootCmd.SetVersionTemplate(banner)
-	rootCmd.PersistentFlags().StringVar(&cfgPath, "config", "/etc/flipt/config/default.yml", "path to config file")
+	rootCmd.PersistentFlags().StringVar(&cfgPath, "config", defaultCfgPath, "path to config file")
```

Comment:
Instead of keeping this flag defaulting to the original path, could we set this to `""` as a default here.
That way if someone happened to explicitly pass --config with the original default path it wouldn't attempt the users config directory first. Which it would currently in this situation.

The logic below would slightly change to only check the defaults paths if the config flag is empty string.

Reply:
@GeorgeMac Makes sense!